AOJ/03/03_D.py

```diff
@@ -0,0 +1,9 @@
+a,b,c=(int(x) for x in input().split())
+i = a
```

Comment:
入力部分以外で変数 `a` は登場していないので、初めから `i` で受け取ると良いと思います。
```suggestion
i, b, c=(int(x) for x in input().split())
```